### PR TITLE
Add AlertDialog theme

### PR DESCRIPTION
**Pull request by onlymash**
Fix click effect of dialog buttons under night theme

### CODE REVIEW

**Comment by Mygod** [comment]
Thanks! I have not tested it but shouldn't this be fixed upstream, i.e. at [material-components-android](https://github.com/material-components/material-components-android)?

**Comment by Mygod** [comment]
Tested this patch and it seems like it actually screws up the color.

**Comment by onlymash** [comment]
you mean background color?, customization is easy

**Comment by Mygod** [comment]
I don't see how your patch fixes click effect either. I looked a bit and material-components-android uses `ThemeOverlay` so that you don't need to do this kind of customization. Do you want to update your patch?

File: plugin/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -24,4 +24,7 @@
         <item name="android:windowCloseOnTouchOutside">false</item>
         <item name="windowNoTitle">true</item>
     </style>
+    <style name="AlertDialogTheme" parent="Theme.MaterialComponents.DayNight.Dialog.Alert">
```
Comment (Mygod) [inline plugin/src/main/res/values/styles.xml]: `ThemeOverlay` should probably be used instead to not screw up the color scheme.

**Comment by Mygod** [comment]
Closing due to inactivity.

**Pull request closed without merging**